test/unit/verifier/lint/test_precommit.py

```diff
@@ -0,0 +1,184 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
```

Comment:
Same as above. Needs to be marked as `... Red Hat ...`.

Reply:
Done!